### PR TITLE
Enable user to run 'python -m jukebot converse --verbose' to display intent confidences

### PR DESCRIPTION
**Pull request by owanr**
I couldn't figure out a way to show the confidences (intent probabilities) while conversing, and it would be great if we had that option.

### CODE REVIEW

**Comment by minhtuev** [comment]
@owanr : can you add in a comment the expected result of your change?

File: mindmeld/components/dialogue.py [hunk 1]
```diff
@@ -1339,6 +1345,7 @@ def process(self, text, params=None, force_sync=False):
             context=self.context,
             frame=self.frame,
             history=self.history,
+            verbose=self.verbose
```
Comment (vijay120) [inline mindmeld/components/dialogue.py:1348]: Change looks good. Can you also add the verbose flag here: https://github.com/cisco/mindmeld/pull/164/files#diff-c21e7ac893d7b0e1d68df2df2ce7ff52R1389

Comment (owanr) [reply mindmeld/components/dialogue.py:1348]: Sorry, I'm not quite sure which line you're referring to. The link doesn't do anything when I click it. Can you please clarify?

Comment (vijay120) [reply mindmeld/components/dialogue.py:1348]: I mean line 1377 (old code) and 1384 (new code) of `mindmeld/components/dialogue.py`: 
```
        response = await self._app_manager.parse(
            text,
            params=internal_params,
            context=self.context,
            frame=self.frame,
            history=self.history,
        )
```

Comment (owanr) [reply mindmeld/components/dialogue.py:1348]: Done! Thank you for the clarification.

**Comment by vijay120** [comment]
@owanr Awesome, thanks for this PR. I am merging this in.

**Pull request merged**